src/super_gradients/recipes/dataset_params/cifar10_dataset_params.yaml

```diff
@@ -1,2 +1,54 @@
-batch_size: 256 # batch size for trainset
-val_batch_size: 512 # batch size for valset in DatasetInterface
\ No newline at end of file
+-batch_size: 256 # batch size for trainset
```

Comment:
Why the "-" ?

Reply:
Was there originally, will remove if it doesn't break tests